### DISCUSSION TITLE
net::err_cert_common_name_invalid

### DISCUSSION THREAD

**Discussion by GianlucaFelli** [vercel/community#4112] [Help] [answered]
### Summary

Hello everyone, i'm working on a website and 'till yesterday was all good, after deploying everithing was fine.
Today instead afte the deploy i received this error " net::err_cert_common_name_invalid " on my chrome browser or any other browser even from mobile.
I don't know what happened and i rellay hope someone can help me asap !

![immagine](https://github.com/vercel/community/assets/133655989/09994af2-1634-450c-9318-8282cde8cb08)
![immagine](https://github.com/vercel/community/assets/133655989/243c63cf-c90a-4a2a-b1e1-29f53995c2ce)
 hoping this screenshot can be usefull many thanks!

### Example

https://www.thebillboard.live/

### Steps to Reproduce

there is no way to replicate i believe cuz actually i don't i know what happened

**Comment by swarnava** [comment, marked as answer]
Hi,

That's likely DNS propagation issue which seems fixed. Are you still seeing the error?

**Reply by GianlucaFelli** [reply]
unfortunatly yes, i mean, probably is my provider because i changed dns on my pc and usign on one pc the firebas's dns and the google's one on the another and it works, but if i try to visit by my phone connected via wifi, it says the connection is not private it treats it like a phishing website. So i'm assuming my provider did something, but there is a way to avoid this problem completely? i mean this website, once finished and completely online, will store a huge quantity of pictures and data the users uploads and it needs to be visible24/7 with no errors at all. I hope it will be repared soon and  there will be no other problmes like that in future ( p.s. sorry for my english mistakes ). Many thanks !

**Reply by swarnava** [reply]
Please make sure your system date and time are correct. You can raise a ticket at https://vercel.com/help
